.github/workflows/update-index.yaml

```diff
@@ -0,0 +1,25 @@
+name: Update index.yaml
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update-index:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+        with:
+          fetch-depth: 0
+      - name: Generate index.yaml
+        uses: mikefarah/yq@master
+        with:
+          cmd: yq ea '[.]' packages/**/package.yaml | yq '... comments=""' | yq '{"packages":.}' > packages/index.yaml
+      - name: Commit index.yaml
```

Comment:
why not simply use git?

https://stackoverflow.com/questions/57921401/push-to-origin-from-github-action/58393457#58393457

Reply:
Now that I think about it… would it make sense for the Workflow to create a PR instead? That way we could prevent having a broken index file caused by a bad workflow.

Reply:
migrated to git and and will now create a pr